packages/gatsby-page-utils/src/create-path.js

```diff
@@ -1,7 +1,6 @@
-// @flow
 const path = require(`path`)
 
-module.exports = (filePath: string): string => {
+module.exports = filePath => {
```

Comment:
```suggestion
module.exports = (filePath) => {
```
is this enforced anywhere?

Reply:
This doesn't appear to be enforced using linting rules anywhere. This PR at this stage passes linting rules. That said, I'll make this change all-the-same